### DISCUSSION TITLE
量算垂直距离，在指定按钮上调用toggleAction('height')方法画垂直距离，选择点后没反应了

### DISCUSSION THREAD

**Discussion by l406054531** [zouyaoji/vue-cesium#402] [Q&A]
<vc-viewer ref="viewerRef" :shouldAnimate="true" :selectionIndicator="false" :infoBox="false" :sceneModePicker="true"
      :orderIndependentTranslucency="false" :sceneMode="3" :contextOptions="contextOptions" @ready="viewerReady">
      <vc-measurements ref="measurementsRef">
        <template #body>
        </template>
      </vc-measurements>
    </vc-viewer>
    <el-button @click="measurementsRef.toggleAction('height')">按钮</el-button>
     const measurementsRef = ref()

**Comment by zouyaoji** [comment]
刚测了没问题，检查一下你的是不是最新版
![image](https://github.com/zouyaoji/vue-cesium/assets/24430116/70adee93-e5a2-4559-b0a2-c3be1da48b9a)

**Comment by l406054531** [comment]
https://codepen.io/l406054531/pen/dyQvpad
再看这个问题，我给距离量算单位配置成了KM，在vc-measurements中的工具测量出来的单位是对的上KM的，但是在外部按钮调用方法测量出来的单位还是默认的M
![image](https://github.com/zouyaoji/vue-cesium/assets/67196408/c5beeb99-df52-4ed1-b37b-7a5e9c3c89da)

**Reply by zouyaoji** [reply]
你定义为km的是distance，而toggleAction是polyline
